README.md

```diff
@@ -208,6 +208,11 @@ See [Dompdf\Options](src/Options.php) for a list of available options.
    [W3C Markup Validation Service](http://validator.w3.org)).
  * Table cells are not pageable, meaning a table row must fit on a single page.
  * Elements are rendered on the active page when they are parsed.
+ * Embedding "raw" SVG's (`<svg><path...></svg>`) isn't working yet, you need to use a DataURI, like this:
+     ```php
+     $html = '<img src="data:image/svg+xml;base64,' . base64_encode($svg) . '" ...>';
```

Comment:
I don't know that we need a sample of data URI embedding in the README. A limitations page on the wiki might be a good place for this.

Either way, we should say either a DataURI or an external SVG document.

Reply:
An own wiki page for those ~8 lines? Given the popularity of SVG related issues I'd rather say it should be visible upfront.

> Either way, we should say either a DataURI or an external SVG document.

Done :-)

Reply:
ha ha I didn't mean just for this issue but for all known issues to provide more detail and work-arounds where possible but this is fine.